rbi/kata-agent/readme.md

```diff
@@ -0,0 +1,5 @@
+# kata-agent的RB
+
+## RB 实现途径
+标准代码是`stable2.1`版本，加上给出的patch，得到的`kata-agent`的sha256值为
```

Comment:
If you provide Chinese version of doc, please rename the file name to readme_cn.md. In addition, the English doc must be provided.